streams/src/main/java/org/apache/kafka/streams/state/internals/RocksDBStore.java

```diff
@@ -124,6 +124,7 @@ public void openDB(ProcessorContext context) {
         options.setCreateIfMissing(true);
         options.setErrorIfExists(false);
         options.setInfoLogLevel(InfoLogLevel.ERROR_LEVEL);
+        options.setIncreaseParallelism(Runtime.getRuntime().availableProcessors());
```

Comment:
The current implementation in https://github.com/facebook/rocksdb/blob/master/util/options.cc is :
DBOptions* DBOptions::IncreaseParallelism(int total_threads) {
  max_background_compactions = total_threads - 1;
  max_background_flushes = 1;
  env->SetBackgroundThreads(total_threads, Env::LOW);
  env->SetBackgroundThreads(1, Env::HIGH);
  return this;
}